Makefile

```diff
@@ -0,0 +1,57 @@
+generator:=ruby
+openapi-generator-cli:=java -jar build/openapi-generator-cli.jar
+services:=balancePlatform checkout legalEntityManagement management payments payouts platformsAccount platformsFund platformsHostedOnboardingPage platformsNotificationConfiguration transfers
+smallServices:=balanceControlService binlookup dataProtection recurring storedValue terminal
+
+binlookup: spec=BinLookupService-v52
+checkout: spec=CheckoutService-v70
+dataProtection: spec=DataProtectionService-v1
+storedValue: spec=StoredValueService-v46
+terminal: spec=TfmAPIService-v1
+payments: spec=PaymentService-v68
+recurring: spec=RecurringService-v68
+payouts: spec=PayoutService-v68
+management: spec=ManagementService-v1
+legalEntityManagement: spec=LegalEntityService-v2
+balancePlatform: spec=BalancePlatformService-v1
+platformsAccount: spec=AccountService-v6
+platformsFund: spec=FundService-v6
+platformsNotificationConfiguration: spec=NotificationConfigurationService-v6
+platformsHostedOnboardingPage: spec=HopService-v6
+transfers: spec=TransferService-v3
+balanceControlService: spec=BalanceControlService-v1
+
+$(services): build/spec
+	wget https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/6.0.1/openapi-generator-cli-6.0.1.jar -O build/openapi-generator-cli.jar
```

Comment:
Thinking maintenance: Should move this url and the cli version to 2 separate variables? that way we dedupe and make it easier to upgrade

Reply:
Sure, good catch will add it.